chatbot-core/api/config/loader.py

```diff
@@ -16,6 +19,12 @@ def load_config():
     """
     config_path = os.path.join(os.path.dirname(__file__), "config.yml")
     with open(config_path, "r", encoding='utf-8') as f:
-        return yaml.safe_load(f)
+        config = yaml.safe_load(f)
+
+    config["is_test_mode"] = os.getenv("IS_TESTING", "0") == "1"
```

Comment:
Now we introduce another way to set the config. It will confuse another developer. You may need to write a specific YAML file for testing and override `config.yml` when running in CI.

Reply:
As workaround, you may write the test config file, and load (read the file) it too. You may set the test config file as highest priority when it's available. So we don't need to copy the file. It's a common practice when you want to have a test config.